app/components/Image/index.tsx

```diff
@@ -21,7 +21,7 @@ const Image = ({
     caption,
     format,
     key,
-  },
+  } = {} as ImageType,
```

Comment:
Non-blocker but,

I read the ticket and tried those report ids, console didn't throw an error at me if I didn't set a default to the image

if the image is possibly undefined, then the type would probably have to change
```
type ImageProps = {
  image?: ImageType;
  ...
```

it's probably less work to destructure the image not in the props, but just like so 

```
const Image = ({
  image,
  height = 0,
  width = 0,
  showTitle = false,
  showCaption = false,
  isZoomable = true,
}: ImageProps): JSX.Element => {
  const {
    data, title, caption, format, key,
  } = image;

...
```

Reply:
I'd still have to add a check for if the image exists in that example though right?

Reply:
yep, I shoulda have illustrated it clearer

```
const { data, title, caption, format, key, } = image ?? {};
``` 
in this case for the default